src/components/Board.js

```diff
@@ -16,14 +16,56 @@ class Board extends Component {
     };
   }
 
+  componentDidMount() {
+    this.getCards()
+  }
+
+  getCards = () => {
+    axios.get('https://inspiration-board.herokuapp.com/boards/Naheed/cards')
+    .then((response) => {
+      this.setState({ cards: response.data });
+    })
+    .catch((error) => {
+      this.setState({ error: error.message });
+    });
+  }
+
+  addCard = (text, emoji) => {
+    axios.post(`https://inspiration-board.herokuapp.com/boards/Naheed/cards?text=${text}&emoji=${emoji}`)
+    .then((response) => {
+      this.getCards()
```

Comment:
instead of making another get request via `this.getCards`, we would prefer that you modify state